VAMobile/e2e/tests/VeteranStatusCard.e2e.ts

```diff
@@ -0,0 +1,134 @@
+import { by, device, element, expect, waitFor } from 'detox'
+import { setTimeout } from 'timers/promises'
+
+import { checkImages, loginToDemoMode, openBenefits, openDisabilityRating, openMilitaryInformation, openPersonalInformation, openProfile } from './utils'
+
+export const VeteranStatusCardConstants = {
+  VETERAN_STATUS_TEXT: 'Proof of Veteran status',
+  VETERAN_STATUS_NAME_TEXT: 'Kimberly Washington',
+  VETERAN_STATUS_MILITARY_BRANCH_TEXT: 'United States Coast Guard',
+  VETERAN_STATUS_DISABILITY_RATING_TEXT: '100% service connected',
+  VETERAN_STATUS_PERIOD_OF_SERVICE_BRANCH_1_TEXT: 'United States Army',
+  VETERAN_STATUS_PERIOD_OF_SERVICE_PERIOD_1_TEXT: 'July 13, 1970 – August 31, 1998',
+  VETERAN_STATUS_PERIOD_OF_SERVICE_BRANCH_2_TEXT: 'United States Coast Guard',
```

Comment:
Since the last branch in the list will always be the same as `VETERAN_STATUS_MILITARY_BRANCH_TEXT`, we probably don't need 2 constants for this